app/views/data_interactive/export.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+
+<div id="json_data">
+  <h3>JSON Query</h3>
+  <form class="form-horizontal" id="transformation_form">
+    <div class="form-group">
+      <textarea name="json-textarea" id="json-textarea" class="form-control top-buffer" data-bind="value: ko.toJSON($data, null, 4)" rows="20"></textarea>
```

Comment:
Out of curiosity, what is `data-bind` doing here? I'm not sure about `$data` variable. JSON can be provided somehow, so the value of this field is automatically populated?